packages/babel-plugin-transform-typescript/test/fixtures/optimize-const-enums/custom-values-exported/options.json

```diff
@@ -0,0 +1,3 @@
+{
+	"BABEL_8_BREAKING": true
+}
```

Comment:
Although it won't affect the results, it seems like this test should enable `optimizeConstEnums`.